openapi-cli/src/test/resources/ballerina-to-openapi/expected_gen/response/byte.yaml

```diff
@@ -0,0 +1,61 @@
+openapi: 3.0.1
+info:
+  title: PayloadV
+  version: 0.0.0
+servers:
+  - url: "{server}:{port}/payloadV"
+    variables:
+      server:
+        default: http://localhost
+      port:
+        default: "9090"
+paths:
+  /challenges/{challenged}:
+    get:
+      operationId: getChallengesChallenged
+      parameters:
+        - name: challenged
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        "200":
+          description: Ok
+          content:
+            application/octet-stream:
+              schema:
+                type: array
+                items:
+                  type: string
+                  format: uuid
```

Comment:
This how `byte[]` return type is mapped to a OpenAPI response in the current implementation. Is this fine ? @NipunaRanasinghe @lnash94

Reply:
@lnash94 is there any particular reason not to use type `string` with format `byte` to represent byte arrays?

Reply:
+1 for the `string` type with format `byte` since the Ballerina byte maps to the base64 encode and the swagger byte also represents it.

Reply:
Updated